### PR TITLE
Split check testlist to enable multiple jobs in CI build

### PR DESCRIPTION
**Pull request by liuguo09**
In github action free version, up to 20 jobs could be run to speed up
CI build.

Signed-off-by: liuhaitao <liuhaitao@xiaomi.com>

### CODE REVIEW

File: cibuild.sh [hunk 1]
```diff
@@ -174,11 +174,11 @@ function run_builds {
   local ncpus=`grep -c ^processor /proc/cpuinfo`
   local options="-j $ncpus"
 
-  if [ "X$build" = "Xcheck" ]; then
+  if [ "X$build" != "Xfull" ]; then
```
Comment (xiaoxiang781216) [inline cibuild.sh]: Should we remove -b and support pass the mulitple file here?

File: cibuild.sh [hunk 2]
```diff
@@ -174,11 +174,11 @@ function run_builds {
   local ncpus=`grep -c ^processor /proc/cpuinfo`
   local options="-j $ncpus"
 
-  if [ "X$build" = "Xcheck" ]; then
```
Comment (xiaoxiang781216) [inline cibuild.sh]: should we pass -x from command line instead of the hardcode relateship between full.data and -x?

File: cibuild.sh (hunk 1 above)
Comment (liuguo09) [reply cibuild.sh]: Ok, done

File: cibuild.sh (hunk 2 above)
Comment (liuguo09) [reply cibuild.sh]: Good suggestion. Done

**Pull request merged**